extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/TaskCreate.java

```diff
@@ -1,51 +0,0 @@
-/**
- * Copyright (C) 2015 Red Hat, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.fabric8.tekton.api.examples;
-
-import io.fabric8.tekton.client.DefaultTektonClient;
-import io.fabric8.tekton.client.TektonClient;
-import io.fabric8.tekton.pipeline.v1beta1.Task;
-import io.fabric8.tekton.pipeline.v1beta1.TaskBuilder;
-import io.fabric8.tekton.pipeline.v1beta1.TaskList;
-
-public class TaskCreate {
```

Comment:
I can still see v1beta1 Task in generated resources. Why do we need to delete this example?

Reply:
I tried to run these and they didn't work. I thought it makes sense to clean up broken examples. I can have another look at this if you want.

Reply:
I restored this file